mimic/test/test_loadbalancer.py

```diff
@@ -236,6 +236,105 @@ def test_lb_status_change_against_undefined_clb(self):
         r = self._patch_attributes_request(lb_id_offset=1000)
         self.assertEqual(r.resp.code, 404)
 
+    def test_update_node_status(self):
+        """
+        `PUT ../loadbalancers/lb_id/nodes/node_id/status` will update node's
```

Comment:
1. I think you mean ``` `PUT .../` ```; `..` means "up one level" in URL language, whereas `...`, i.e. an ellipsis, means "some prefix, and then" in English.
2. _actually_ I think you mean ``` ``PUT ...`` ``` because single backticks mean "reference" in ReST and double-backticks mean "monospaced text".  We need an automatic PR builder for docs which uploads artifacts so I can point at the malformed text :-\.